README.md

```diff
@@ -69,6 +81,49 @@ Okta's Spring Security integration will [parse the JWT access token](https://dev
 Check out a minimal example that uses the [Okta Signin Widget and JQuery](examples/siw-jquery) or [this blog post](https://scotch.io/tutorials/build-a-secure-notes-application-with-kotlin-typescript-and-okta). 
```

Comment:
Change this link to the version on our blog: https://developer.okta.com/blog/2017/09/19/build-a-secure-notes-application-with-kotlin-typescript-and-okta